plumbing/util.go

```diff
@@ -115,39 +114,6 @@ func ExecSilentlyToWriter(cmdString string, targetDir string, writer io.Writer)
 
 }
 
-func ExecToLogWithProgressDots(cmdString string, targetDir string, logPath string) error {
```

Comment:
This was mostly a "mark" to remind us that we should use some better terminal handling library/tool to show some progress. I've added something in the remote build code path for it, but it's not optimal.

Reply:
Understood, but it's unused code and not properly synchronized. Rather than fixing it, I'm opting to remove the code.

Context: staticcheck is warning that a `for/select/default` combo can cause busy-waits. This is true in general, but the `default` case includes a `time.Sleep`. However, the code doesn't use a `time.Ticker` or other such to permit `stoppedchan` to interrupt the sleep, so the goroutine may leak past the end of the overall function.